opennms-dao-api/src/main/java/org/opennms/netmgt/dao/api/OutageDao.java

```diff
@@ -64,6 +64,11 @@
      */
     OnmsOutage currentOutageForService(OnmsMonitoredService service);
 
+    /**
+     * Return all current open outages for the given service be it detected from local or from remote (perspective).
+     */
+    Collection<OnmsOutage> currentOutagesForService(OnmsMonitoredService service);
```

Comment:
This is confusing. Could we make this `currentPerspectiveOutagesForService`?

Reply:
@fooker why do you think this is confusing? It gives all outages back - remote or local.

Reply:
The function itself is correct. I found it confusing that it is only used to show outages with perspectives due to filtering for perspective location everywhere it is used.

Reply:
fixed.